cms/templates/index.html

```diff
@@ -485,6 +500,54 @@ <h3 class="title">${_('Create Your First Library')}</h3>
       </div>
       %endif
 
+      % if is_programs_enabled:
+        % if len(programs) > 0:
+          <div class="programs programs-tab">
+            <!-- Classes related to courses are intentionally reused here, to duplicate the styling used for course listing. -->
+            <ul class="list-courses">
+              % for program in programs:
+              <li class="course-item">
+
+                <!-- TODO: Use the program ID contained in the dict to link to the appropriate view in the authoring app. -->
+                <a class="program-link" href="#">
+                  <h3 class="course-title">${program['name'] | h}</h3>
+
+                  <div class="course-metadata">
+                    <span class="course-org metadata-item">
+                      <!-- As of this writing, programs can only be owned by one organization. If that constraint is relaxed, this will need to be revisited. -->
+                      <span class="label">${_("Organization:")}</span> <span class="value">${program['organizations'][0]['key'] | h}</span>
```

Comment:
@andy-armstrong I'm seeing separate "label" / "value" spans throughout the template, structured the same as this one.  Since labels and values have their own spans I would have assumed this would work with rtl (but, i'm no expert, and I didn't write this).

Can you point me toward an example of a more optimal way to do this?  Is it worth adopting a different approach without updating the entire template?